runtime/dart_init.cc

```diff
@@ -78,18 +78,18 @@ const char kSnapshotAssetKey[] = "snapshot_blob.bin";
 
 namespace {
 
-static const char* kDartMirrorsArgs[] = {
+// Arguments passed to the Dart VM in all configurations.
+static const char* kDartLanguageArgs[] = {
     "--enable_mirrors=false",
+    "--background_compilation",
+    "--await_is_keyword",
+    "--warn_super",
```

Comment:
I'm not sure we should be enforcing warn_super at runtime.  We already have it at analysis time, which seems proper.  If we enforce it at runtime, we won't be able to run code from packages that fail this check.  If it's going to be part of strong mode, let's turn on runtime enforcement together with the rest of the dart ecosystem then.